docs/references/express/available-methods.mdx

```diff
@@ -0,0 +1,89 @@
+---
+title: Express.js Available Methods
+description: Learn how to use the Clerk Express.js SDK to interact with the Clerk API.
+---
+
+# Express.js Available Methods
```

Comment:
ℹ️ The Clerk Express SDK exports the following:
- `@clerk/backend` exports (re-exports)
- `clerkMiddleware`
- `getAuth`
- `requireAuth`

Should we list them in the "Available methods" section and describe how they can be used?

PS: we probably need to mention that the `clerkMiddleware` is required for the `getAuth` and `requireAuth` to work.

cc: @alexisintech